### DISCUSSION TITLE
More accurate Time prayed

### DISCUSSION THREAD

**Discussion by squigglybob** [Pray4Movement/prayer-global-porch#44] [Ideas]
At the moment this is just a best guess, as it is based on the user's chosen amount of time that they want to pray for.

E.g. a user is using the default 1min timer and prays for 20 places, they will be recorded as having prayed for 20 mins, even if they actually spent 5 mins praying for each place.

Should/Could we instead, track the time between them entering into a prayer experience for a place, and clicking next/done?

This would give a more accurate picture of time actually spent praying, but would also be an upper limit of time prayed, rather than the current lower limit of time prayed.

What do you think @ChrisChasm ?

**Comment by ChrisChasm** [comment]
The idea currently on this is to in the background develop a continuing timer looking for additional minutes on the prayer page, and submit updates to the prayer log record to increment the value per each minute on the page.

https://www.loom.com/share/a6ff9eb290174722be6b7f87c2960a35
